### PR TITLE
2.3

### PR DESCRIPTION
**Pull request by ThomasLandauer**

### CODE REVIEW

File: book/doctrine.rst [hunk 1]
```diff
@@ -951,8 +951,6 @@ To relate the ``Category`` and ``Product`` entities, start by creating a
                 products:
                     targetEntity: Product
                     mappedBy: category
-            # don't forget to init the collection in the __construct() method
```
Comment (WouterJ) [inline book/doctrine.rst]: I think this comment is kind of usefull (and not everyone uses `doctrine:generate:entities`, even more it's better to never use this command).

**Comment by WouterJ** [comment]
I like the other changes! (btw, can you please remove your merge commits from the commit history?)

File: book/doctrine.rst (hunk 1 above)
Comment (ThomasLandauer) [inline book/doctrine.rst]: OK, then please change it to:
# The ArrayCollection 'products' is automatically initialized in the __construct() method, when you call doctrine:generate:entities

**Comment by ThomasLandauer** [comment]
Sorry, I'm new to GitHub and don't know where to delete what :-(
Took me a while to even figure out that I did not effectively create the Pull Request at the time I wrote the changes ;-)

**Comment by stof** [comment]
@WouterJ you can squash the commits with the GH tool anyway, so this is not a hard requirement

**Comment by WouterJ** [comment]
@ThomasLandauer no worries, I'll delete them when merging in that case.

**Comment by ThomasLandauer** [comment]
Thanks :-)

File: book/forms.rst [hunk 2]
```diff
@@ -97,7 +97,7 @@ from inside a controller::
                 ->getForm();
 
             return $this->render('default/new.html.twig', array(
-                'form' => $form->createView(),
+                'form' => $form->createView()
```
Comment (xabbuh) [inline book/forms.rst]: This should be reverted to comply with the Symfony CS.

File: book/templating.rst [hunk 3]
```diff
@@ -370,6 +370,19 @@ When working with template inheritance, here are some tips to keep in mind:
           {{ parent() }}
       {% endblock %}
 
+* Blocks can be nested. For better overview, you can add the block name to the
+  ``{% endblock %}`` tag like so:
```
Comment (xabbuh) [inline book/templating.rst]: I would say "like this".

File: cookbook/form/form_customization.rst [hunk 4]
```diff
@@ -282,7 +282,7 @@ can now re-use the form customization across many templates:
 
 .. code-block:: html+jinja
 
-    {# app/Resources/views/Form/fields.html.twig #}
+    {# src/AppBundle/Resources/views/Form/fields.html.twig #}
```
Comment (stof) [inline cookbook/form/form_customization.rst]: this looks wrong to me. Best practices are to put templates in ``app/Resources/views``

Comment (ThomasLandauer) [inline cookbook/form/form_customization.rst]: Anyhow. My point is that the path used in the code exampled throughout this page is inconsistent! I don't know what the best practice is. If you don't want to change it here, then please change it in the next codeblock (and probably some others).

**Comment by WouterJ** [comment]
Hi @ThomasLandauer!

I've taken your commits and added a new commit to revert some changes (but I kept most of them!) and fixed the template name inconsistency. I put these commits in a new PR: https://github.com/symfony/symfony-docs/pull/6050

So I'm going to close this PR, but your contributions are not lost and you still get the credits. 

Thanks for helping us improving the documentation, I'm amazed you discovered these errors and took time to fix them! :christmas_tree:

**Comment by ThomasLandauer** [comment]
Thanks :-)

**Pull request closed without merging**